scripts/mod_loader/modui/mod_configuration.lua

```diff
@@ -149,6 +168,50 @@ local function createUi()
 			return r
 		end
 		
+		local sortDropDownOptions = {
+			GetText("ModConfig_Button_Sort_Choice_1"),
+			GetText("ModConfig_Button_Sort_Choice_2"),
+			GetText("ModConfig_Button_Sort_Choice_3")
+		}
+		
+		--- adds buttons to control sorting
+		local function addSortButtons()
+			-- dropdown to choose a sorting
+			local buttonLayoutRight = UiBoxLayout()
+				:hgap(20)
+				:heightpx(40)
+				:addTo(buttonLayout)
+			buttonLayoutRight.alignH = "right"
+			local presetDropdown = UiDropDown(sortDropDownOptions)
```

Comment:
Might  want to use `UiDropDown(values, strings, defaultValue)` constructor, so that you can compare selection to index, rather than exact strings.

Reply:
Makes sense; in case someone were to localize it.